net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -8,15 +8,15 @@ IPT6="ip6tables -t mangle -w"
 LOG="logger -t mwan3[$$] -p"
 CONNTRACK_FILE="/proc/net/nf_conntrack"
 
-MWAN3_STATUS_DIR="/var/run/mwan3track"
+MWAN3_STATUS_DIR="/var/run/mwan3"
 
+[ -d $MWAN3_STATUS_DIR ] || mkdir $MWAN3_STATUS_DIR $MWAN3_STATUS_DIR/state
```

Comment:
Could you use 
[ -d $MWAN3_STATUS_DIR ] || mkdir **-p** $MWAN3_STATUS_DIR/state